vue-components/src/components/Lookup.vue

```diff
@@ -137,6 +140,15 @@ export default Vue.extend( {
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 			this.$el.querySelector( 'input' )!.blur();
 		},
+		onScroll( firstIndex: number, lastIndex: number ): void {
+
+			if ( firstIndex !== this.scrollIndexStart || lastIndex !== this.scrollIndexEnd ) {
```

Comment:
I think we should add a comment here to describe this event in storybook. See [lines 121-124](https://github.com/wmde/wikit/blob/16466afe3f0794cfef195a4d8d2633f31adde997/vue-components/src/components/Lookup.vue#L120-L124)

Reply:
Forgot about this one. Thanks for the heads up.